### PR TITLE
mkosi: fix some fall-out from the pathification

### PR DESCRIPTION
**Pull request by poettering**
A follow-up from the pathlib.Path conversion (#782)

### CODE REVIEW

**Comment by DaanDeMeyer** [comment]
Lgtm

**Comment by behrmann** [comment]
LGTM

**Comment by behrmann** [comment]
CI error seems unrelated

File: mkosi/__init__.py [hunk 1]
```diff
@@ -6050,7 +6054,7 @@ def load_args(args: argparse.Namespace) -> CommandLineArguments:
         die("Sorry, --verity can only be used with unified kernel images")
 
     if args.source_file_transfer is None:
-        if os.path.exists(".git") or args.build_sources.joinpaths(".git").exists():
+        if os.path.exists(".git") or args.build_sources.joinpath(".git").exists():
```
Comment (keszybz) [inline mkosi/__init__.py:6057]: This was already fixed in 3f2be95a, but having it a second time doesn't hurt.

**Pull request merged**